src/main/java/rife/continuations/instrument/TypesMethodVisitor.java

```diff
@@ -302,6 +333,40 @@ public void visitInsn(int opcode) {
                     currentNode_ = null;
                     break;
                 case ATHROW:
+                case MONITORENTER:
```

Comment:
Please don't rearrange, they are grouped by type of opcode, not by type of result.

Reply:
Duplicate code vs. slight ease while debugging. Sure, why not.